extensions/cornerstone/src/initMeasurementService.js

```diff
@@ -375,7 +375,12 @@ const connectMeasurementServiceToTools = (
           label: data.annotation.data.label,
           frameNumber: frameNumber,
         },
-      });
+      };
+      if (data.annotation.data?.polyline) {
```

Comment:
Can we make this more generic so it works for all graphic types?

Reply:
I changed to put all attributes in data.annotation.data. Is that correct @sedghi?

Reply:
I make this import because here an important attribute, polyline, isn´t imported. That could be the case for another annotations.